packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/iam/iam.html

```diff
@@ -0,0 +1,58 @@
+<div class="text-center" data-ng-if="$ctrl.loading">
+    <oui-spinner></oui-spinner>
+</div>
+
+<div class="row" data-ng-if="!$ctrl.loading">
+    <div class="col-sm-9">
+        <dl>
+            <dd data-translate="dedicatedCloud_USER_iam_description"></dd>
+            <dd>
+                <small
+                    data-translate="dedicatedCloud_USER_iam_usage_warning"
+                ></small>
+            </dd>
+            <dt data-translate="dedicatedCloud_USER_iam_status"></dt>
+            <dd>
+                <ovh-manager-pcc-dashboard-component-activation-status
+                    class="d-block mt-1"
+                    data-status-name=":: $ctrl.iamStatus.state"
+                ></ovh-manager-pcc-dashboard-component-activation-status>
+            </dd>
+        </dl>
```

Comment:
**nitpick:** Inappropriate usage of `dd` & `dt`. Description list must have `dt` and a corresponding `dd`

Reply:
Switched the two first dd to divs